etl/steps/data/garden/climate/2024-02-19/monthly_burned_area.meta.yml

```diff
@@ -0,0 +1,45 @@
+# NOTE: To learn more about the fields, hover over their names.
+definitions:
+  common:
+    presentation:
+      topic_tags:
+        - Climate Change
+    display:
+      numDecimalPlaces: 0
+      zeroDay: '2000-01-01'
+      yearIsDay: true
+    short_unit: ha
+    unit: hectares
+
+
+# Learn more about the available fields:
+# http://docs.owid.io/projects/etl/architecture/metadata/reference/dataset/
+dataset:
+  update_period_days: 365
+
+
+# Learn more about the available fields:
+# http://docs.owid.io/projects/etl/architecture/metadata/reference/tables/
+tables:
+  monthly_burned_area:
+    # Learn more about the available fields:
+    # http://docs.owid.io/projects/etl/architecture/metadata/reference/indicator/
+    variables:
+      forest:
+        title: Monthly burned area across forests
```

Comment:
I would include at least description_short to explain what forest, savannas, shrublands, grasslands, etc are

Reply:
thanks! :)